src/pages/latestprojects/Components/NoteCard.tsx

```diff
@@ -0,0 +1,118 @@
+import { Box, Text, Flex, Link } from '@chakra-ui/react';
+import { useState } from 'react';
+
+const ImageWithReload = ({ src, alt, ...rest }) => {
+  const [error, setError] = useState(false);
+
+  const handleError = () => {
+    setError(true);
+  };
+
+  return (
+    <img
+      src={error ? '' : src}
+      alt={alt}
+      onError={handleError}
+      style={{ width: '120px', height: '120px', maxHeight: '120px', objectFit: 'cover' }}
+      {...rest}
+    />
+  );
+};
+
+interface Props {
+  content: string;
+  nostrich: string;
+}
+
+export default function NoteCard({ nostrich, content, date }: Props) {
+  let parsedContent;
+  try {
+    parsedContent = JSON.parse(content);
+  } catch (error) {
+    console.error('Error parsing JSON:', error);
+    parsedContent = {};
+  }
+  const { display_name, image, about, website } = parsedContent;
+
+  return (
+    <Link href={website} textDecoration="none" _hover={{ textDecoration: 'none' }}>
+      <Flex
+        className="notecard"
+        alignItems="flex-start"
+        overflow="hidden"
+        margin="16px 0"
+        paddingTop="0px"
+        border="2px solid #ccc"
+        borderRadius="8px"
+        backgroundColor="neutral.0"
+        transition="border-color 0.5s"
+        boxShadow="none"
+        flexWrap="wrap"
+        _hover={{ cursor: 'pointer', borderColor: 'neutral.400' }}
+        _active={{ borderColor: 'primary.400' }}
+        _focus={{ borderColor: 'primary.400' }}
+      >
+        <Box
+          flex={{ base: '0 0 120px', md: '0 0 auto' }}
+          minWidth="120px"
+          position="relative"
+          overflow="hidden"
+          marginLeft="0px"
+          marginRight={{ base: '0px', md: '20px' }}
+          marginBottom={{ base: '0px', md: '0px' }}

```

Comment:
maybe user lg? it doesnot make sense to change at md, our point of change for mobile mode is lg.